libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/MediaUtils.java

```diff
@@ -379,6 +379,33 @@ public static String getMediaFileName(File mediaFile, String mimeType) {
         return originalFileName;
     }
 
+    /**
+     * Given the full file path, or the filename with extension (i.e. my-picture.jpg), returns the filename part only (my-picture).
+     *
+     * @param filePath The path to the file or the full filename
+     * @return filename part only or null
+     */
+    public static String getFileNameFromPath(String filePath) {
```

Comment:
Would `FileUtils.java` be a better place for this method, since it's not strictly related to media?

Reply:
Right, will change it.